crypto/ec/ec_mult.c

```diff
@@ -236,6 +238,21 @@ int ec_scalar_mul_ladder(const EC_GROUP *group, EC_POINT *r,
         }
     }
 
+    /*
+     * Create random values for BN_consttime_swap_randomized, and for
+	 * constant-time swapping of Z_is_one in EC_POINT_SWAP below
+	 * This randomizatin is needed to prevent side channel attacks
+     * (for details, see: "Nonce@Once: A Single-Trace EM Side Channel Attack on
+     *  Several Constant-Time Elliptic Curve Implementations in Mobile Platforms"
+     * by M. Alam, B. Yilmaz, F. Werner, N. Samwel, A. Zajic, D. Genkin, Y. Yarom,
+     * and M. Prvulovic, in IEEE Euro S&P 2021)
+     */
+    BN_ULONG *swap_rand=OPENSSL_malloc((cardinality_bits+2)*sizeof(BN_ULONG));
+    if(RAND_bytes_ex(bn_get_libctx(ctx),(unsigned char *)swap_rand,(cardinality_bits+2)*sizeof(BN_ULONG))<=0) {
```

Comment:
`bn_get_libctx` is called `ossl_bn_get_libctx` in the latest master so this is causing the CIs to fail.

Reply:
Code style: spaces around operators